spec/index.html

```diff
@@ -2466,7 +2457,8 @@ <h3>Examples</h3>
         <h3>Property Paths and Equivalent Patterns</h3>
         <p>SPARQL property paths treat the RDF triples as a directed, possibly cyclic, graph with
           named edges. Some property paths are equivalent to a 
-          <a href="#sparqlTranslatePathExpressions">translation</a> into triple patterns and SPARQL UNION graph
+          <a href="#sparqlTranslatePathExpressions">translation</a>
+          into triple patterns and SPARQL UNION graph
```

Comment:
```suggestion
          into triple patterns and SPARQL `UNION` graph
```

Reply:
The text is technically wrong because of entailment extensions. UNION does not always work.

Removing that and the whole sentence can be removed - it's covered by examples above).

The examples did get fixed during 1.1 (no UNION), this was missed.